dist/systemd/system/keylime_agent.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=The Keylime compute agent
+
+[Service]
+ExecStart=/usr/bin/keylime_agent
```

Comment:
Do we want to specify a (non-root) user here?

Reply:
Yes it is a good idea to run keylime not as root by default. Example on what we do for the Debian package:
https://github.com/utkarsh2102/python-keylime/blob/master/debian/python3-keylime-agent.keylime_agent.service

Also the mount for `/var/lib/keylime/secure` can then created by systemd:
https://github.com/utkarsh2102/python-keylime/blob/master/debian/python3-keylime-agent.var-lib-keylime-secure.mount

Reply:
@THS-on: nice! Can't (some of) these changes be upstreamed?

Reply:
@sergio-correia the current installer.sh script just assumes that you run everything as root which is fine because it is mostly used for testing and CI. It is up to the distributions on how to package keylime and if for example they want to create a new user etc.

If there is interest I see no reason why not to include those as an example on how to run Keylime without root.

Reply:
@THS-on @sergio-correia @ueno Was I reading this correctly that we are still waiting for a change on this PR re: a non-root user?

Reply:
No, what the Debian package currently does is not ideal and will not work for packages  actually shipped in distributions. So we can merge it as currently is.

See discussion here: https://github.com/keylime/keylime/issues/864